features/queryselector.yml

```diff
@@ -0,0 +1,6 @@
+name: querySelector
```

Comment:
I think this could also go in the base DOM feature—it's pretty ancient—though there is a caniuse feature for this. I feel less strongly about this one though.